src/components/Report_reader.jsx

```diff
@@ -1,23 +1,36 @@
-function Li_temp() {
+import PropTypes from 'prop-types';
+
+function Li_temp({ text, cost }) {
   return(
     <li className="py-1 px-4 flex items-center justify-between">
-      <span>Finance Fee</span>
-      <span>$140</span>
+      <span className="text-left">{text}</span>
+      <span>${cost.toFixed(2)}</span>
     </li>
   );
 }
 
-export default function ReportReader() {
-  
-  return(
+Li_temp.propTypes = {
+  text: PropTypes.string.isRequired,
+  cost: PropTypes.number.isRequired,
+};
+
```

Comment:
(Optional)
It is working very well. but we have a very tiny problem. As the first, i used temp name for creating each `li`. I named it `Li_temp`. we can change to a good name. To manage our codes, it is better to place our proptypes validation at the end of page.